.github/actions/build-binary/action.yml

```diff
@@ -30,6 +30,9 @@ runs:
     - name: Install cross
       run: cargo install cross
       shell: bash
+    - name: Help cross find the git repo
+      run: cp -r .git/ ./quickwit/.git
```

Comment:
Use a symlink.

Reply:
Yes much better. I just wanted something radical given the time it takes to test it. Also, this ends up mounted in a docker container.

Reply:
Can't we use passthrough env variables for `cross`?
https://github.com/cross-rs/cross#passing-environment-variables-into-the-build-environment

Reply:
Yes, it was used at some point

Reply:
The build script is nop if git is not available in the env. `QuickwitBuildInfo` attempts to read the `QW_COMMIT_*` en vars directly. So as long as they are populated by the build script or whatever process, we're good.